schema.ts

```diff
@@ -392,6 +397,84 @@ export interface ScaleSchema extends BaseNumberSchema<ValueHintType.Scale> {
   icon?: ScaleIconSet;
 }
 
+/**
+ * Display types that can be used with an {@link EmailSchema}.
+ */
+export enum EmailDisplayType {
+  /**
+   * Display both icon and email (default).
+   */
+  IconAndEmail = 'iconAndEmail',
+
+  /**
+   * Display icon only.
+   */
+  IconOnly = 'iconOnly',
+
+  /**
+   * Display email address only.
+   */
+  EmailOnly = 'emailOnly',
+}
+
+/**
+ * A schema representing a return value or object property that is an email address.
+ *
+ * An email address can be represented visually as an icon, an icon plus the email address, or
+ * the just the email address.  Auto-complete against previously typed domain names is
```

Comment:
Total nit, I think we use `autocomplete` without a dash elsewhere in the documentation.